jina/drivers/evaluate.py

```diff
@@ -118,4 +119,57 @@ def _apply_all(self,
                 gt_content = pb2array(gt_content)
             evaluation.value = self.exec_fn(doc_content, gt_content)
             evaluation.op_name = f'{self.metric}-{self.exec.metric}'
-            evaluation.ref_id = groundtruth.id
\ No newline at end of file
+            evaluation.ref_id = groundtruth.id
+
+
+class GroundTruthKVIndexDriver(BaseIndexDriver):
```

Comment:
i dont see the purpose of having a gtkvindexdriver, it is the same with KVindex

Reply:
I agree that I can have a change to have the `BaseKVIndexDriver` access a specific `key`

Reply:
Solved!